proposals/src/header.rs

```diff
@@ -113,6 +115,7 @@ impl TypedChainId {
             | TypedChainId::RococoParachain(id)
             | TypedChainId::Cosmos(id)
             | TypedChainId::Solana(id) => *id,
+            TypedChainId::Ink(id) => *id,
```

Comment:
Please use the same format as the other types since this doesn't seem to require any change.